### PR TITLE
인트로페이지 버튼까지 작업, 아이콘은 아직입니다.

### PR DESCRIPTION
**Pull request by chepchep2**

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#17** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/17?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#16** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/16?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#15** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/15?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#14** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/14?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#13** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/13?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/12?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#11** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/11?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#10** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/10?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#9** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/9?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#8** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/8?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/7?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#6** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/6?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/5?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/4?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/3?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/2?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/common/auth_button.dart [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+    required this.backgroundColor,
+    required this.text,
+    this.borderColor,
+    this.textColor,
+  });
 
   @override
   Widget build(BuildContext context) {
-    return const Placeholder();
+    return ElevatedButton(
```
Comment (flowkater) [inline lib/common/auth_button.dart:24]: ElevatedButton 을 쓰지 말고,
그냥 DecoratedBox 랑 GestureDetector 를 쓰는 방향으로 가면 좋을 것 같습니다.

```dart
GestureDetector(
        onTap: widget.onTap,
        child: DecoratedBox(
          decoration: BoxDecoration(
            borderRadius: BorderRadius.circular(32.0),
            border: Border.all(
              color: Colors.black,
            ),
            color: _isHovered ? Colors.black : Colors.white,
          ),
          child: Padding(
            padding: const EdgeInsets.symmetric(
              horizontal: 24.0,
              vertical: 18.0,
            ),
            child: Text(
              widget.buttonText,
              style: TextStyle(
                color: _isHovered ? Colors.white : Colors.black,
                fontSize: 16.0,
              ),
            ),
          ),
        ),
      ),
```

ElevatedButton 같은 기본 버튼은 머터리얼 스타일에 종속적이라서 권장하지 않는 방식입니다.

**Review by flowkater — CHANGES_REQUESTED** [review]
위 상황 한번 반영해주시면 좋을 것 같습니다.

**Comment by flowkater** [comment]
### Merge activity

* **Mar 7, 8:27 AM EST**: @flowkater started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/2).
* **Mar 7, 8:28 AM EST**: @flowkater merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/2).

**Pull request merged**